README.md

```diff
@@ -579,6 +580,314 @@ Note that the subflows are an array of repeating entries - one for each iteratio
 subflow. Each iteration has a unique UUID associated with it so we can have a way of working with a
 specific iteration's data.
 
+## PDF Generation
+
+The Form Flow Library uses the [Apache PDFBox](https://pdfbox.apache.org/) library to generate PDFs
+from user input. In order to begin generating PDFs from your user responses, you will need to first
+create a template PDF file with prepared fields and a `pdf-map.yaml` file that will act as a map of
+inputs to PDF fields.
+
+### Creating a template PDF file
+
+The first step in generating PDFs is to create a template PDF file. This file will be used by the
+FFB
+to generate a PDF with user responses mapped to specific PDF fields. We recommend using Adobe
+Acrobat and
+the [Prepare Form Tool](https://www.adobe.com/acrobat/resources/how-to-create-fillable-pdf.html).
+
+When preparing your fields in Adobe Acrobat, in the field properties window, you will want to give
+each field a unique name.
+
+For text fields, this name should be different for each field. For example,
+given the PDF needs mappings for a first name field and last name field, names for each field could
+be `APPLICANT_FIRST_NAME` and `APPLICANT_LAST_NAME`.
+
+For radio fields, while the name should be unique, each radio will have the same name but different
+`Radio Button Choice` values within the `Options` tab of the field properties window. These
+`Radio Button Choice` values should match the actual values the radio inputs in your application.
+For example, a PDF with a radio field for `Gender` could have three radio buttons with the name
+Gender,
+and three separate values of `Male`, `Female`, and `Non-binary` matching the values of the
+corresponding
+input in the application.
+
+For checkbox fields, each checkbox should have a unique name, but the `Export Value` within the
+fields properties window under the `Options` tab should be `Yes`. This is because the FFB will map
+the
+actual value of the checkbox in the application to `Yes`, meaning that box will be checked if that
+value was selected by the user. For example, a PDF with a checkbox field for `programs` could have
+three checkboxes with the names `SNAP`, `TANF`, and `CCAP`, where the export value for each
+is `Yes`.
+
+### Creating a pdf-map.yaml file
+
+The next step in generating PDFs is to create a `pdf-map.yaml` file. This file will act as a map of
+input fields from your application to PDF fields in your template PDF file. This file should be
+added
+to your applications `resources` folder, in a folder named `pdfs`.
+
+Your `pdf-map.yaml` file will consist of the following key value pairs:
```

Comment:
Instead of leaving this as a `:`, it might be better to reword this, like so:
`The following sections describe the expected keys in the `pdf-map.yaml` file.`